frontend/www/js/omegaup/components/course/CardsList.test.ts

```diff
@@ -56,12 +108,39 @@ const coursesListProps = {
 };
 
 describe('CardsList.vue', () => {
-  it('Should handle empty courses list for user', () => {
-    const wrapper = shallowMount(course_CardsList, {
-      propsData: coursesListProps,
+  it('Should list public courses', async () => {
+    const wrapper = mount(course_CardsList, {
+      propsData: {
+        courses: coursesListProps.courses,
+        type: 'public',
+      },
     });
 
-    expect(wrapper.find('.public').text()).toContain(T.courseListPublicCourses);
-    expect(wrapper.find('.student').text()).toContain(T.courseListIStudy);
+    expect(wrapper.text()).toContain(publicCoursesTitles[0]);
+    expect(wrapper.text()).toContain(publicCoursesTitles[1]);
+
+    expect(wrapper.find('div.dropdown-divider').element.style.display).toBe(
+      'none',
+    );
```

Comment:
ya con https://github.com/omegaup/omegaup/pull/5108 debería poderse usar

```suggestion
    expect(wrapper.find('div.dropdown-divider').element).not.toBeVisible();
```

sin el hack del import! así ya no debería ser necesario checar ningún estilo.